app/views/example_sentences/_list.html.haml

```diff
@@ -0,0 +1,7 @@
+- if word.example_sentences.present?
+  = two_column_card t('words.show.example_sentences.title'), t('words.show.example_sentences.description') do
+    = box padding: false, class: 'striped' do
+      - word.example_sentences.each do |sentence|
+        .p-4
```

Comment:
```suggestion
        .p-4.flex.flex-wrap.justify-between
```

Reply:
This might have been lost in the rebase?

Reply:
Wow, well spotted, Serge! Yes, that was los in the rebase conflict.